tensorflow/core/ops/logging_ops.cc

```diff
@@ -80,6 +80,9 @@ REGISTER_OP("ImageSummary")
     .Input("tag: string")
     .Input("tensor: T")
     .Output("summary: string")
+    .Attr("vmin: float = 9999.99")
```

Comment:
This default value may not be large enough; some images have 16 bits per channel per pixel.